resources/jargon.md

```diff
@@ -2,14 +2,37 @@ Here are some terms, jargon, and abbreviations that are used in the technology w
 
 # Contribution
 
-Instructors, TAs, staff, volunteers, and students: if you come across a term you don't know, please look it up and submit a PR adding it to this list!  
+Instructors, TAs, staff, volunteers, and students: if you come across a term you don't know, please look it up and submit a PR adding it to this list!
 
 Please keep the list in alphabetical order.
 
 # Jargon
 
+- **argument**: An input passed to a function in a call to it, related to parameter. A function can take zero, one or many arguments. Most languages also provide a way to pass a variable number of arguments. 
+
+- **command line interface**: A text based interface to the operating system. Programs or commands are invoked by typing them. There is a long and rich history of using these simple programs and even using them in combination. A rough synonym for console. 
+
+- **console**: A text based interface to the operating system. A rough synonym for command line. 
+
 - **dictionary**: A synonym for an [associative data structure](https://en.wikipedia.org/wiki/Associative_array), which maps keys to values.
 
 - **foo**: A common placeholder name used for variables and functions when the actual name isn't important. See [foobar](https://en.wikipedia.org/wiki/Foobar).
 
+- **high level**: 
+    1. An overview with less detail. Ex., "We're going to discuss the requirements on a high level before we get into the implementation."
+    1. When used to describe languages and technology it often indicates a simpler means of doing something.
+
+- **holy war**: An unsolvable debate about some aspect of computing that has strong adherents in more than one camp. It's good to recognize when an issue is one of style or choice and when it's being presented otherwise. 
```

Comment:
Might be nice to give some examples, such as iOS vs Android, or vim vs emacs vs Sublime?